src/city/components/CitiesTable.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import {
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow
+} from "@mui/material";
+
+export const CitiesTable = () => {
+
+    return (
+        <>
```

Comment:
Why do you use here React Fragment <> here?

Reply:
I left them because in next PR I added edit modals to component, but here (in this PR) they are unnecessary

Reply:
Why do you use here React Fragment <> here?